generator/src/test/java/com/scottlogic/deg/generator/cucumber/steps/DateValueStep.java

```diff
@@ -33,12 +34,45 @@ public void producedDataShouldContainTemporalValuesForField(String fieldName){
         helper.assertFieldContainsNullOrMatching(fieldName, LocalDateTime.class);
     }
 
+    @Then("{fieldVar} contains anything but temporal data")
+    public void producedDataShouldContainAnythingButStringValuesForField(String fieldName){
+        helper.assertFieldContainsNullOrNotMatching(fieldName, LocalDateTime.class);
+    }
+
     @Then("{fieldVar} contains temporal values between {date} and {date} inclusively")
     public void producedDataShouldContainTemporalValuesInRangeForField(String fieldName, DateObject minInclusive, DateObject maxInclusive){
         helper.assertFieldContainsNullOrMatching(
             fieldName,
             LocalDateTime.class,
-            value -> isAfterOrAt(value, minInclusive) && isBeforeOrAt(value, maxInclusive));
+            isBetweenInclusively(minInclusive, maxInclusive));
+    }
+
+    @Then("{fieldVar} contains temporal values outside {date} and {date}")
```

Comment:
Then description could be inclusive or exclusive. Should we be more specific?

Reply:
Parameters renamed from `xxxInclusive` to `xxx` (they will be exclusive, but including the suffix _Exclusive_ then passing them to `xxxInclusively()` would be wierd!